src/setup_node_env/harden/console.js

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+// From https://www.ascii-code.com/characters/control-characters,
+// but explicitly allowing the range \u0008-\u000F (line breaks, tabs, etc.)
+var CONTROL_CHAR_REGEXP = new RegExp('[\\u0000-\\u0007\\u0010-\\u001F]', 'g');
+
+var unsafeConsole = {
+  // debug: console.debug.bind(console),
+  // error: console.error.bind(console),
+  // info: console.info.bind(console),
+  log: console.log.bind(console),
+  // trace: console.trace.bind(console),
+  // warn: console.warn.bind(console),
+};
```

Comment:
Even if probably widely unused, the `console` object exposed more methods than this: https://developer.mozilla.org/en-US/docs/Web/API/console

We don't care about exposing them on the `unsafeConsole` though given we know which subset of APIs we will be using, so it's probably fine.

Reply:
Agreed. I will double check all of our use cases to make sure they are covered.